include/cucumber-cpp/internal/drivers/GenericDriver.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef CUKE_GENERICDRIVER_HPP_
+#define CUKE_GENERICDRIVER_HPP_
+
+#include "../step/StepManager.hpp"
+
+namespace cucumber {
+namespace internal {
+
+class FakeStep : public BasicStep {
```

Comment:
You've mentioned only class name in #80 - wasn't sure if this should also be changed